.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,8 +8,12 @@ List of open PRs: https://github.com/NixOS/nixpkgs/pulls
 Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
 -->
 
-###### Motivation for this change
+###### Description of this change
 
+<!--
+For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
+For new packages please briefly describe and provide a link to the package homepage, this helps additional maintainers find your package.
```

Comment:
Can you rework that sentence? I'm not sure what to "describe". Also I'm not sure how "find your package" is a good argument.